packages/toolkit/package.json

```diff
@@ -86,7 +69,43 @@
     "webpackbar": "^5.0.2"
   },
   "devDependencies": {
-    "glob": "^7.2.0"
+    "@10up/babel-preset-default": "^2.0.1",
+    "@10up/eslint-config": "^2.4.4",
+    "@10up/stylelint-config": ">= 1.0.0",
+    "eslint": "^8.0.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jest": "^25.2.4",
+    "eslint-plugin-jsdoc": "^37.0.3",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.27.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "glob": "^7.2.0",
+    "prettier": "^2.4.1",
+    "stylelint": "^13 || ^14"
+  },
+  "peerDependencies": {
+    "@10up/babel-preset-default": "^2.0.0",
+    "@10up/eslint-config": "^2.0.0",
+    "@10up/stylelint-config": ">= 1.0.0",
+    "eslint": "^8.0.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jest": "^25.2.4",
+    "eslint-plugin-jsdoc": "^37.0.3",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.27.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-webpack-plugin": "^3.1.1",
```

Comment:
eslint-webpack-plugin is not required by `@10up/eslint-config`. I think we can keep it as a regular dep of toolkit.